tests/plugins/test_add_filesystem.py

```diff
@@ -213,7 +213,42 @@ def test_add_filesystem_plugin_generated(tmpdir, docker_tasker, scratch):
         workflow,
         [{
             'name': AddFilesystemPlugin.key,
-            'args': {'koji_hub': KOJI_HUB}
+            'args': {
+                'koji_hub': KOJI_HUB,
+                'from_task_id': task_id,
+                'architecture': 'x86_64'
+            }
+        }]
+    )
+
+    results = runner.run()
+    plugin_result = results[AddFilesystemPlugin.key]
+    assert 'base-image-id' in plugin_result
```

Comment:
Good idea asserting the plugin result!
Think we're missing checking the value for `'filesystem-koji-task-id'`.
You can simplify these assertion statements to something like this:
```python
expected_plugin_result = {
    'base-image-id': IMPORTED_IMAGE_ID,
    'filesystem-koji-task-id': FILESYSTEM_TASK_ID,
}
assert plugin_result == expected_plugin_result
```